### PR TITLE
Adding test that verifies for 4369 to be fixed

### LINKED ISSUE #4369: HPX freezes up with multiple new_ calls in a simple test program
The simple test program, available here:
https://github.com/dmarce1/new_test
when given the argument "4", should allocate 4096 bare bones components across 4 nodes. It freezes up when attempting to to this on queenbee. When running the same program with argument "3", it successfully allocates 512 components in less than a second and exits.

## Steps to Reproduce the Problem
Run the test program available here:
https://github.com/dmarce1/new_test
on 4 nodes of queenbee with argument 4.

## Specifications

... Please describe your environment

HPX commit 258bf04aae03c6864e06a0fad0f5c5848310a300
GCC 8.3 
queenbee2

### PR DESCRIPTION
**Pull request by hkaiser**
- flyby: clean up CMakeFile.txt for this test

Fixes #4369

### CODE REVIEW

File: tests/regressions/component/CMakeLists.txt [hunk 1]
```diff
@@ -1,25 +1,27 @@
...
     create_n_components_2323
     moveonly_constructor_arguments_1405
-    new_2848
     returned_client_2150
    )
 
 if(HPX_WITH_NETWORKING)
   set(tests ${tests}
-      create_remote_component_2334
+    bulk_new_3054
```
Comment (msimberg) [inline tests/regressions/component/CMakeLists.txt]: These don't make sense with networking off?

Comment (hkaiser) [reply tests/regressions/component/CMakeLists.txt]: Let me have a look again. Yu might be right, they most likely perform local tests as well.

**Comment by hkaiser** [comment]
This was still missing some changes (see comments above). I'll create a new PR for those.

File: tests/regressions/component/CMakeLists.txt (hunk 1 above)
Comment (hkaiser) [reply tests/regressions/component/CMakeLists.txt]: This has been addressed by #4415

**Pull request merged**